modules/imaging_qc/test/TestPlan.md

```diff
@@ -0,0 +1,15 @@
+# Imaging Quality Control - Test Plan
+1. Access Imaging Quality Control page, ensure that it renders.
+2. Make sure this module can be viewed if and only if the user has “Quality Control access” permission.
+3. Make sure this module only loads if the `mri_parameter_form` table exists and properly configured.
```

Comment:
```suggestion
3. Make sure this module only loads if the `mri_parameter_form` table exists and properly configured.
```

If the module loads and the `mri_parameter_form` doesn't exist, I don't think that's a bug.

Reply:
The README for this module specifies that this table is necessary for the module to load. The idea was that if it loads anyway, the module is not functioning as intended and may be loading because of an oversight.

Reply:
I just skimmed the README, but it looks like it's specifiying the prerequisites for using the module, not saying those are intended limitations of how the module should work.